### ISSUE TITLE
Files are sorted case sensitively.

### ISSUE THREAD

**Issue by internalfx** [internalfx/regrid#15]

**Comment by internalfx** [comment]
@bchavez have any thoughts on this?

**Comment by bchavez** [comment]
No thoughts really; other than we haven't really had a bug in the C# driver / ReGrid code in a really, *really*, long time. IIRC, there hasn't been any complaints with ReGrid on this particular topic.

**Comment by internalfx** [comment]
True, I also don't know if it's been used in any serious capacity though.

When I retrieved a list of files, the sorting was not what I was expecting or wanting. And there is no easy way to change it.

My thoughts were that case sensitive sorting is contrary to how Linux, OSX, and (I believe) Windows sort files.
